src/explorer/CosmosDBTreeItem.ts

```diff
@@ -5,22 +5,23 @@
 
 import { StringDictionary } from 'azure-arm-website/lib/models';
 import * as vscode from 'vscode';
-import { ISiteTreeRoot, validateAppSettingKey } from 'vscode-azureappservice';
+import { IAppSettingsClient, ISiteTreeRoot, validateAppSettingKey } from 'vscode-azureappservice';
 import { AzExtTreeItem, AzureParentTreeItem, GenericTreeItem, ICreateChildImplContext, UserCancelledError } from 'vscode-azureextensionui';
 import { AzureExtensionApiProvider } from 'vscode-azureextensionui/api';
 import { ext } from '../extensionVariables';
 import { nonNullProp } from '../utils/nonNull';
 import { getThemedIconPath, IThemedIconPath } from '../utils/pathUtils';
 import { CosmosDBExtensionApi, DatabaseTreeItem } from '../vscode-cosmos.api';
-import { ConnectionsTreeItem } from './ConnectionsTreeItem';
 import { CosmosDBConnection } from './CosmosDBConnection';
+import { SiteTreeItem } from './SiteTreeItem';
 
 export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
     public static contextValueInstalled: string = 'сosmosDBConnections';
     public static contextValueNotInstalled: string = 'сosmosDBNotInstalled';
     public readonly label: string = 'Azure Databases';
```

Comment:
Let's make this label just "Databases". It's already a very "Azure" scenario, so I don't think that's necessary. I think that's what Matt asked for in the email thread, too